agent.md

```diff
@@ -1,46 +1,76 @@
 Installing Agent for Hosted Kubecost (Alpha)
-======
+============================================
 
-The `kubecost-agent` is a version of the kubecost metric exporter that runs as a primary transport of data into hosted kubecost. In order to install the `kubecost-agent`, you will need a specific storage key provided by the kubecost team.
+The ___kubecost-agent___ is a lightweight Kubecost exporter that sends metrics to hosted kubecost. In order to install the ___kubecost-agent___, you will need a specific key provided by the kubecost team.
 
 The name of the storage key file provided by the kubecost team will have the name `kubecost-agent.key`
 
-## Installation From Helm
-1. Add the kubecost helm repository: 
+## Installation Using Helm
+
+> Note: integration with CI/CD tools is certainly possible, but we recommend following this guide as closely as possible to ensure a successful deployment.
+
+1. Add the kubecost helm repository:
 ```bash
 helm repo add kubecost https://kubecost.github.io/cost-analyzer/
 ```
 
 2. The following will install the kubecost agent and required components using the provided `kubecost-agent.key` (ensure the key file is in the current file directory):
 ```bash
-helm install kubecost kubecost/cost-analyzer \
-  --namespace kubecost \
+helm upgrade --install kubecost kubecost/cost-analyzer \
+  --namespace kubecost-agent --create-namespace\
```

Comment:
Is there a reason we are changing this to include `-agent`? Are people attempting to install both cost-model and the agent so we need a separate namespace?

Reply:
Just to make it consistent with the documentation we provided to our most recent customer. 

Good call out, I'm not tied to -agent.

Though, for a customer, it makes it obvious that this is an "agent"?

Reply:
Yeah, sounds like @mbolt35 is on board with this change as well, and it's really his call when it comes to hosted stuff